src/com/lftechnology/java/training/niraj/ems/enums/Status.java

```diff
@@ -0,0 +1,25 @@
+package com.lftechnology.java.training.niraj.ems.enums;
+
+public enum Status {
+    ACTIVE("1"), INACTIVE("0");
+
+    private final String IS_ACTIVE = "Yes";
+    private final String IS_INACTIVE = "No";
+    private final String isActive;
+
+    private Status(String isActive) {
+        this.isActive = isActive;
+    }
+
+    public String getStatus() {
+        return isActive;
+    }
+
+    public String getStatusString() {
+        if (isActive.equals("1")) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "1" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1132)